### DISCUSSION TITLE
Qexo云端命令库已上线

### DISCUSSION THREAD

**Discussion by am-abudu** [Qexo/Qexo#279] [Announcements]
位于 设置->云端命令
详情可见 https://github.com/Qexo/Scripts 命令可使用 functions.py 的内置函数

**Comment by Fgaoxing** [comment]
@am-abudu 感觉接口还是太少

**Reply by am-abudu** [reply]
你还需要什么接口

**Comment by Fgaoxing** [comment]
而且，print是对前端还是后端？

**Reply by am-abudu** [reply]
可以参考hello-world.py
